subgraphs/harvest-finance/src/utils/vaults.ts

```diff
@@ -177,6 +177,11 @@ export namespace vaults {
 
     protocol.totalPoolCount = protocol.totalPoolCount + 1
 
+    // workaround: issue with derivedFrom
+    protocol._vaults = protocol._vaults
+      ? protocol._vaults!.concat([vault.id])
```

Comment:
Concatenating [String] to an array of strings. Should be replaced for  protocol._vaults!.concat(vault.id)